maintainers/scripts/nix-guix-correlate.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/python
```

Comment:
Can you make this a package instead? Also `/usr/bin/python` would never work directly on nixos.